helm/linkerd2-app/templates/config.yaml

```diff
@@ -1 +1,33 @@
-# config.yaml is moved to hooks/hook-pre-install-config.yaml
```

Comment:
since the certs aren't in the configmap anymore we can move it back and track upstream more easily.